R/pkg/inst/tests/testthat/test_mllib.R

```diff
@@ -387,6 +387,11 @@ test_that("spark.mlp", {
   model <- spark.mlp(df, layers = c(4, 5, 4, 3))
   mlpPredictions <- collect(select(predict(model, mlpTestDF), "prediction"))
   expect_equal(head(mlpPredictions$prediction, 10), c(1, 1, 1, 1, 0, 1, 2, 2, 1, 0))
+
+  # Test illegal parameter
+  expect_error(spark.mlp(df, layers = NULL))
```

Comment:
it would be preferred to add the error string to check for (the message in stop()) - please see our other tests on how to do this.